lib/apiForSheets.js

```diff
@@ -165,7 +240,11 @@ module.exports.checkClaStatus = async function (contextData) {
 
   // Authorize a client with the loaded credentials.
   const oauthClient = module.exports.authorize(JSON.parse(clientSecret));
+  const oauthClientWIthOnBoardingSecret = module.exports.
+    authorize(JSON.parse(OnBoardingSheetId));
 
   // Call the sheets API with the authorized client.
   module.exports.checkClaSheet(oauthClient);
```

Comment:
Should this still be here?
I believe the `checkClaAndOnboardingTracker` handles this as well.

Reply:
removed it as we are utilizing same `oauthClient` for accessing both sheets.